src/framework/accessibility/internal/accessibilitycontroller.cpp

```diff
@@ -154,6 +154,8 @@ void AccessibilityController::propertyChanged(IAccessible* item, IAccessible::Pr
         break;
     case IAccessible::Property::Name: etype = QAccessible::NameChanged;
         break;
+    case IAccessible::Property::Description: etype = QAccessible::DescriptionChanged;
+        break;
```

Comment:
@shoogle: is that something you just forgot to add?

Reply:
Probably. In #8912 I just did the minimum necessary to get it to compile and run. I'll try to pay more attention to warnings next time as well as errors.

Reply:
This wasn't a complaint, more the request for an official blessing of that change ;-)